src/components/ListView/ListView.test.js

```diff
@@ -0,0 +1,321 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import {
+  fireEvent,
+  waitFor,
+} from '@testing-library/react';
+
+import { createMemoryHistory } from 'history';
+
+import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+import '../../../test/jest/__mock__';
+import { translationsProperties } from '../../../test/jest/helpers';
+
+import { ListView } from './ListView';
+
+const history = createMemoryHistory();
+
+history.push = jest.fn();
+
+jest.mock('../ViewContainer/getCRUDActions', () => ({
+  getCRUDActions: () => ({
+    onCreate: () => 'success!',
+    onEdit: () => 'success!',
+    onDelete: () => 'success!',
+    onRestoreDefaults: () => 'success!',
+  }),
+}));
+jest.mock('@folio/stripes/components', () => ({
+  ...jest.requireActual('@folio/stripes/components'),
+  ConfirmationModal: jest.fn(({
+    open,
+    onCancel,
+    onConfirm,
+  }) => (open ? (
+    <div>
+      <span>Confirmation modal</span>
+      <button
+        type="button"
+        onClick={onCancel}
+      >
+        Cancel
+      </button>
+      <button
+        type="button"
+        id="confirmButton"
+        onClick={onConfirm}
+      >
+        Confirm
+      </button>
+    </div>
+  ) : null)),
+}));
+
+const testSet = new Set(['testId1']);
+
+const listViewProps = listViewFilterProps => {
+  return {
+    ...listViewFilterProps,
+    selectedRecord: {
+      record: {
+        id: 'testId1',
+        parentProfiles: 'testParent',
+        childProfiles: 'testChild',
+      },
+      hasLoaded: true,
+    },
+    setList: jest.fn(),
+    RecordView: jest.fn(),
+    history: { push: history.push },
+  };
+};
+
+const listViewPropsActionProfiles = {
+  resources: {
```

Comment:
Use `buildResources` and `buildMutator` helpers from `@folio/stripes-data-transfer-components/test/helpers`